src/System.Private.CoreLib/src/System/Environment.Unix.cs

```diff
@@ -84,5 +84,10 @@ public static unsafe string MachineName
                 return Encoding.UTF8.GetString(hostName, hostNameLength);
             }
         }
+
+        public static void Exit(int exitCode)
+        {
```

Comment:
Binder calls the non-portable `RhpShutdown` that does a PInvoke transition which is why I asked about that in my above comment. Is bypassing that okay?